src/presentation/controller/addressLookUp/Routing.ts

```diff
@@ -218,7 +218,20 @@ const addressRoutingTerritoryChoiceGeneralPartnerCorrespondanceAddress = {
   }
 };
 
-const correspondenceAddress = [addressRoutingTerritoryChoiceGeneralPartnerCorrespondanceAddress];
+const addressRoutingPostcodeGeneralPartnerCorrespondenceAddress = {
+  previousUrl: url.TERRITORY_CHOICE_GENERAL_PARTNER_CORRESPONDENCE_ADDRESS_URL,
+  currentUrl: url.POSTCODE_GENERAL_PARTNER_CORRESPONDENCE_ADDRESS_URL,
```

Comment:
I know the next page selected from territory choice is dynamic, but in keeping with what we've done in the other routing config (e.g. `addressRoutingTerritoryChoiceGeneralPartnerUsualResidentialAddress` above), I think the `nextUrl` on **line 214** (so not this line) should be set to url.POSTCODE_GENERAL_PARTNER_CORRESPONDENCE_ADDRESS_URL and not `LIMITED_PARTNERS_URL`.

Reply:
The page is not there yet, but the url is so could change this.

Reply:
Thought I'd done this, as it was navigating to this page, made sure it's present now.